### ISSUE TITLE
[BUG] Exporting via reporting on dark mode broke

### ISSUE THREAD

**Issue by kavilla** [opensearch-project/dashboards-reporting#111]
@BT-93 commented on [Wed May 03 2023](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3979)

When dark mode is set and you export to either PNG or PDF the background of the exports is white.
This was not an issue prior to version 2.6 (I think).

**To Reproduce**
1. Go to Advanced Settings
2. Search for Dark Mode
3. Set the slider to Dark Mode
4. Go to a dashboard
5. Select Reporting
6. Go either PNG or PDF
7. Exported doco background is white

**Expected behavior**
The background to be color #141519

**OpenSearch Version**
2.6 & 2.7

**Dashboards Version**
2.6 & 2.7

**Plugins**
The default ones

**Screenshots**
![image](https://user-images.githubusercontent.com/94328741/236122444-27957bf3-783c-47aa-bead-a4896fd659da.png)

**Host/Environment (please complete the following information):**
 - OS: Windows 10
 - Browser and version: Chrome Version 113.0.5672.64 (Official Build) (64-bit)

**Additional context**
:)

---

@zhongnansu commented on [Thu May 04 2023](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3979#issuecomment-1535351005)

@opensearch-project/triage Please transfer this issue to reporting repo https://github.com/opensearch-project/dashboards-reporting

**Comment by wbeckler** [comment]
I would propose that reporting should always export on a pure white background, because it is used for printing and emailing stuff for other people to print.

**Comment by dagneyb** [comment]
@wbeckler - can we give the customer the option (default to report mirroring their dashboards settings, with an option to "print mode with white background" or something? I think given that most people don't print things anymore and are probably emailing reports, they likely actually want them to look how they have set their Dashboards to look. @kgcreative @mnkugler - thoughts?

**Comment by AMoo-Miki** [comment]
When the user chooses to export a report, a new window is opened with the actual dashboard, visualization, or ..., the "source", and a screenshot of the page is taken and wrapped in a PDF. As a result, it will always match the color-scheme applied to OSD for the tenant the user is using. 

If we wanted to allow users to choose a theme for exporting:
* Technically, it is possible to inject a different theme's styling but for that the plugin will have to know every styling that OSD, the visualizations, and perhaps even other plugins would have added to render the different theme; that information is not readily available and is probably going to be a challenge to gather and make available to plugins.
* When the plugin opens the new window to the "source", it can ask OSD to override the color-scheme via a query-string parameter. Such a mechanism doesn't exist but wouldn't be tough to create. However, because this plugin cannot close the window it opened and the new window is completely usable, the user will be left with a bunch of windows using the dark color-scheme and a bunch that use the light color-scheme.

In short, as long as the report is generated by the browser, we are better off leaving the export to match the user's chosen color-scheme, and if selection of theme is an option we want to offer, the choices are either technically prohibitive or detract from the user experience.

**Comment by mnkugler** [comment]
Thanks Miki! That makes sense and agree that we are better off leaving the export to match the user's chosen color-scheme.
